cupy/_logic/content.py

```diff
@@ -39,7 +41,57 @@ def _create_float_test_ufunc(name, doc):
     ''')
 
 
-# TODO(okuta): Implement isneginf
-
-
-# TODO(okuta): Implement isposinf
+def isneginf(x, out=None):
+    """Returns a bool array, where True if input element is negative infinity.
+    Args:
+        x (cupy.ndarray): Input array.
+    Returns:
+        cupy.ndarray: Boolean array of same shape as ``x``.
+    Examples
+    --------
+    >>> cupy.isneginf(0)
+    False
+    >>> cupy.isneginf([4, -4, numpy.inf, -numpy.inf])
+    [False, False, False, True]
```

Comment:
Please make an example appropriate (i.e., examples should use cupy.ndarray as Args section says it is a cupy.ndarray)

Reply:
This seems unresolved.

Reply:
Done!